spark-connector/v3.3/spark-runtime/build.gradle.kts

```diff
@@ -11,13 +11,21 @@ plugins {
 }
 
 val scalaVersion: String = project.properties["scalaVersion"] as? String ?: extra["defaultScalaVersion"].toString()
-val sparkVersion: String = libs.versions.spark.get()
+val sparkVersion: String = libs.versions.spark33.get()
 val sparkMajorVersion: String = sparkVersion.substringBeforeLast(".")
 val baseName = "${rootProject.name}-spark-connector-runtime-${sparkMajorVersion}_$scalaVersion"
 
 dependencies {
   implementation(project(":clients:client-java-runtime", configuration = "shadow"))
-  implementation(project(":spark-connector:spark-connector"))
+  when (sparkMajorVersion) {
+    "3.3" -> {
+      val kyuubiVersion: String = libs.versions.kyuubi4spark33.get()
+      println("Applying Spark 3.3 dependencies")
```

Comment:
We can remove this line, this is useless.

Reply:
ok